torch/fx/experimental/symbolic_shapes.py

```diff
@@ -18,6 +18,19 @@
 
 SYM_FUNCTION_MODE = None
 
+math_lib = torch.library.Library("math", "DEF")
+
+math_lib.define("mul(int a, int b) -> int")
+math_lib.define("eq(int a, int b) -> bool")
+math_lib.define("gt(int a, int b) -> bool")
+math_lib.define("lt(int a, int b) -> bool")
```

Comment:
FWIW, JIT "prim" actually already defines operators that look like these. They don't have dispatcher entries though.

Reply:
How come these don't have SymInt in their schema?